### PR TITLE
Completed css-four-column-layout challenge. Bryan Valdovinos

### PR DESCRIPTION
**Pull request by bryanvaldovinos**
Screen Shot 2022-05-16 at 9 55 50 PM

### CODE REVIEW

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="row center">
+    <h2>How It Works</h2>
+  </div>
+  <div class="row center">
+    <div class="col-10">
+      <!-----EMPTY-->
+    </div>
```
Comment (codingcodymiller) [inline css-four-column-layout/index.html:20]: You shouldn't ever have empty columns just to take up space. You can get this behavior by just adding a class with `justify-content: center` to the `row` class parent element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-four-column-layout/index.html (hunk 1 above)
Comment (bryanvaldovinos) [reply css-four-column-layout/index.html:20]: ah okay! good to know moving forward, empty columns removed. thanks Cody!

**Comment by bryanvaldovinos** [comment]
Screen Shot 2022-05-17 at 12 56 31 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Comment by bryanvaldovinos** [comment]
TY sir!

**Pull request merged**